### DISCUSSION TITLE
Is it possible to configure JSON Logging for AKHQ?

### DISCUSSION THREAD

**Discussion by wilkejo** [tchiotludo/akhq#1060] [Q&A]
Hey,

Is there an option / possibility to configure JSON Logging for AKHQ? 
Our Logging Infrastructure is configured to handle JSON Logs as best.

Thank you

**Comment by tchiotludo** [comment]
You will need to provide `JAVA_OPTS` env var with `-Dlogback.configurationFile=/path/to/config.xml` with a logback configuration that log in json format

**Comment by wilkejo** [comment]
Thank you for the answer. I spoke with a Java Developer and he said he is missing a dependency for a log appender in json. Maybe you had an other approach.
Do you mind providing a possible example logback config?

**Comment by theBNT** [comment]
Just created a PR to describe how to achieve JSON logging: https://github.com/tchiotludo/akhq/pull/1772
